miasm2/expression/expression.py

```diff
@@ -1136,7 +1177,11 @@ def __new__(cls, *args):
         return Expr.get_object(cls, args)
 
     def __str__(self):
-        return '{' + ', '.join(["%s %s %s" % (arg, idx, idx + arg.size) for idx, arg in self.iter_args()]) + '}'
+        return '{' + ', '.join(["%s %s %s" % (arg, idx, idx + arg.size) for
```

Comment:
The `[` in `([` is useless here; it wil create an array and then iter over it, instead of an iterator